README.md

```diff
@@ -54,6 +54,63 @@ Example configuration:
     ```
     See the [Kafka Connect transforms](https://kafka.apache.org/31/documentation.html#connect_transforms) documentation for more details.
 
+**Note:** For each message record that the sink connector writes to Cloudant the document ID from this priority order:
```

Comment:
Wording seems a bit clumsy. How about something like "The ID of each document written to Cloudant by the sink connector can be configured as follows..."